realis.go

```diff
@@ -277,9 +278,13 @@ func NewDefaultClientUsingCluster(cluster *Cluster, user, passwd string) (Realis
 		return nil, err
 	}
 	fmt.Printf(" url: %s\n", url)
+	var secure bool
```

Comment:
It is shorter to just say

```
secure := strings.HasPrefix(url, "https")
```

Reply:
oops Im doing that anyways. Will change that.